src/sender.ts

```diff
@@ -66,6 +67,8 @@ export class Sender {
 
   private sequence = 0;
 
+  private resendStatus = false;
```

Comment:
I think it would be reasonable to make this property public, since we're exposing it via events anyway

Reply:
fixed in a0294d60cd5aefcf28fc322e63674fc456661fd6